src/lib/mapper.js

```diff
@@ -38,6 +43,33 @@ export default class Mapper {
 
   }
 
+  // execute(source, destination) {
```

Comment:
It won't stay. I was in the middle of a refactor when you raised the bug. Just wanted to get it to you as quick as I could.